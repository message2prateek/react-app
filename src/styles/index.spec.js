import * as React from 'react'

import { shallow } from 'enzyme'

import GlobalStyle from './'

describe('GlobalStyle', () => {
  it('matches the snapshot', () => {
    expect(toJson(shallow(<GlobalStyle />))).toMatchSnapshot()
  })
})
