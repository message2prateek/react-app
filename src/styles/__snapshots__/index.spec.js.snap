// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GlobalStyle matches the snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
